app/controllers/api/v1/targets_controller.rb

```diff
@@ -0,0 +1,14 @@
+module Api
+  module V1
+    class TargetsController < Api::V1::ApiController
+      def create
+        authorize Target
+        @target = TargetService.new(current_user, resource_params).create!
```

Comment:
Do you really need a service just for create instead of using directly active record here? 🤔

Reply:
Yep you are right, thanks! Done!